spec/type.dd

```diff
@@ -73,6 +75,71 @@ $(H2 $(LEGACY_LNAME2 Base Types, base-types, Base Types))
 enum E : T { ... }  // T is the base type of E
 ---
 
+$(H2 $(LNAME2 Unsigned Type, Unsigned Type))
+
+    $(P The following are $(I Unsigned Type):)
+
+    $(UL
+        $(LI bool)
+        $(LI ubyte) $(LI ushort) $(LI uint) $(LI ulong) $(LI ucent)
+        $(LI char) $(LI wchar) $(LI dchar)
+    )
+
+    $(P If the $(GLINK2 enum, EnumBaseType) of an enum is an $(I Unsigned Type), the enum
+    is an $(I Unsigned Type).)
+
+    $(P If the element type of a vector is an $(I Unsigned Type), the vector
+    is an $(I Unsigned Type).)
+
+$(H2 $(LNAME2 Integral Type, Integral Type))
```

Comment:
Should this say "Integral Type, Signed Type"?

Reply:
Perhaps not.

Reply:
Integral includes signed and unsigned types.

Reply:
I got confused when one section said "Unsigned Type" and one said "Integral Type".